1.illumination_correction/README.md

```diff
@@ -0,0 +1,77 @@
+# Cell Profiler Illumination Correction Pipeline
+In this module, I present the 10 CellProfiler pipelines I created using various manual parameters to create the illumination function and correct the raw images. The chosen image for comparison is **D6_01_1_4_DAPI_001.tif** due to the large artifact present in the bottom right corner.
+
+Within the [Images folder](1.illumination_correction/Images), there is only the one image from the data set that I chose to compare with each other. 
```

Comment:
Can you add a comment about how you calculated the illumination function? It's clear to me that you're using only 1 image to compare results, but you're calculating the illumination function using all NF1 images, correct?